diff -rupN libevhtp-1.2.8.orig/CMakeLists.txt libevhtp-1.2.8/CMakeLists.txt
--- libevhtp-1.2.8.orig/CMakeLists.txt	2016-04-22 23:03:16.000000000 +0200
+++ libevhtp-1.2.8/CMakeLists.txt	2016-04-22 23:03:29.487920424 +0200
@@ -55,7 +55,7 @@ if (NOT HAVE_SYS_UN)
 endif(NOT HAVE_SYS_UN)
 
 # -DEVHTP_DISABLE_SSL:STRING=ON
-OPTION(EVHTP_DISABLE_SSL       "Disable ssl support"      OFF)
+OPTION(EVHTP_DISABLE_SSL       "Disable ssl support"      ON)
 
 # -DEVHTP_DISABLE_EVTHR:STRING=ON
 OPTION(EVHTP_DISABLE_EVTHR     "Disable evthread support" OFF)
