diff -rupN Imaging-1.1.7.orig/_imagingft.c Imaging-1.1.7/_imagingft.c
--- Imaging-1.1.7.orig/_imagingft.c	2009-11-01 01:44:12.000000000 +0100
+++ Imaging-1.1.7/_imagingft.c	2016-04-21 14:30:17.224442259 +0200
@@ -70,7 +70,7 @@ struct {
     const char* message;
 } ft_errors[] =
 
-#include <freetype/fterrors.h>
+#include <fterrors.h>
 
 /* -------------------------------------------------------------------- */
 /* font objects */
